website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -246,6 +246,13 @@ To update:
 
 </Tabs>
 
+## Using VS Code extensions
+
+Visual Studio (VS) Code extensions enhance command line tools by adding extra functionalities. The dbt Cloud CLI is fully compatible with dbt Core, however it doesn't support some dbt Core APIs required by certain tools, for example VS Code extensions. 
+
+To use these extensions, such as dbt-power-user, with the dbt Cloud CLI, you can install it using Homebrew (along with dbt Core) and create an alias to run the dbt Cloud CLI as `dbt-cloud`. This allows dbt-power-user to continue to work alongside the dbt Cloud CLI.
```

Comment:
"This allows dbt-power-user to continue to invoke dbt Core under the hood, alongside the dbt Cloud CLI."